dataset/dictionary/characters.json5

```diff
@@ -1680,7 +1680,7 @@
   {
     en: "Nameless",
     ja: "無名",
-    zhCN: "无名",  
+    zhCN: "无名",
```

Comment:
Just FYI, if your editor has EditorConfig plugin and install it, the trailing spaces are automatically deleted on save.
Ignore this if you already know that.

Reply:
Thanks for your advice. The plugin is helpful.